.golangci.yaml

```diff
@@ -64,7 +64,7 @@ linters:
     - errcheck
     - funlen
     - goconst
-    - gocritic
+  # - gocritic
```

Comment:
I assume this is for debugging, but you will be trying to nail down the gocritic issues on the cli/runtime/config?

Reply:
reverted the change